test/action_controller/json_api/pagination_test.rb

```diff
@@ -0,0 +1,116 @@
+require 'test_helper'
+require 'will_paginate/array'
```

Comment:
How we are going to work without `will_paginate/array` ? Because we will need `ActiveRecord` in order to do that.
Could you give me any example, please?